Organization/Organization+CoreDataClass.h

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+#import <CoreData/CoreData.h>
+
+@class Employee;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Organization : NSManagedObject
+
+- (NSInteger)getCountEmployees;
```

Comment:
You shouldn't modify generated classes, since they may be regenerated in the future and you will loose all your changes. Create Entity+Custom extension for that.